test/test_ca.py

```diff
@@ -91,3 +153,162 @@ def test_revoke_cert(local_ca_fixture, tmpdir):
 
     revoked_cert = crl.get_revoked_certificate_by_serial_number(serial_number)
     assert revoked_cert is not None
+
+
+@pytest.mark.ipa
+def test_ipa_server_simple_setup(ipa_meta_client, dummy_ipa_vals, clean_ipa,
+                                 caplog):
+    ipa_ca = CA.IPAServerCA(ip_addr=dummy_ipa_vals["server_ip"],
+                            client_hostname=dummy_ipa_vals[
+                                "client_hostname"],
+                            hostname=dummy_ipa_vals["server_hostname"],
+                            root_passwd=dummy_ipa_vals[
+                                "server_root_passwd"],
+                            admin_passwd=dummy_ipa_vals[
+                                "server_admin_passwd"],
+                            domain=dummy_ipa_vals["server_domain"])
+
+    ipa_ca.setup()
+
+    # Test if meta client can get info about freshly configured host
+    ipa_meta_client.host_show(a_fqdn=dummy_ipa_vals["client_hostname"])
+
+    # Cleanup after test
+    ipa_meta_client.host_del(a_fqdn=dummy_ipa_vals["client_hostname"])
+
+
+@pytest.mark.ipa
+@pytest.mark.parametrize("force", (False, True))
+def test_ipa_server_setup_force(installed_ipa, force, dummy_ipa_vals,
```

Comment:
code duplication with test_ipa_server_simple_setup

Reply:
makes sense. will fix this